cypress.json

```diff
@@ -1,10 +1,10 @@
 {
   "pluginsFile": "tests/e2e/plugins/index.js",
   "video": false,
-  "defaultCommandTimeout": 10000,
+  "defaultCommandTimeout": 30000,
   "execTimeout": 60000,
   "taskTimeout": 60000,
   "pageLoadTimeout": 60000,
-  "requestTimeout": 10000,
+  "requestTimeout": 30000,
```

Comment:
Increased because I noticed some tests on GH actions were taking 6, 7 seconds. And `mutations.js` failed a couple times, first for taking longer than 10 seconds, then because it had the skeleton loader still being displayed. Locally this timeout shouldn't matter, but 30 seconds seems to be good enough for GH actions.